script/utils/parameters/OptimismGoerliParameters.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+address constant OPTIMISM_GOERLI_DEPLOYER =
+    0xc625F59d51ecDff57FEFE535C80d318CA42A0Ec4;
+
+address constant OPTIMISM_GOERLI_KWENTA_ADMIN_DAO_MULTI_SIG =
+    0xc625F59d51ecDff57FEFE535C80d318CA42A0Ec4;
```

Comment:
**Question:** Is this correct? On etherscan this appears to be an EOA.

Reply:
Yes, this is an EOA on etherscan. Its name is just meant to represent what it *should* be on mainnet. I use an EOA here bc it makes making changes on testnet much easier/faster.